src/ConnectedMode/UI/ManageBinding/ManageBindingViewModel.cs

```diff
@@ -138,6 +138,13 @@ public async Task InitializeDataAsync()
         var displayBindStatus = new TaskToPerformParams<AdapterResponse>(DisplayBindStatusAsync, UiResources.FetchingBindingStatusText,
                 UiResources.FetchingBindingStatusFailedText) { AfterProgressUpdated = OnProgressUpdated};
         await ProgressReporter.ExecuteTaskWithProgressAsync(displayBindStatus);
+
+        if (!IsCurrentProjectBound)
+        {
+            var detectSharedBinding = new TaskToPerformParams<AdapterResponse>(CheckForSharedBindingAsync, UiResources.CheckingForSharedBindingText,
+                UiResources.CheckingForSharedBindingFailedText) { AfterProgressUpdated = OnProgressUpdated };
```

Comment:
Not sure if the `UiResources.CheckingForSharedBindingFailedText` is ever displayed, I couldn't test or figure out how to trigger it.

Reply:
It is shown when an error occurs during the TaskToPerform is executed. So if you throw an exception in the CheckForSharedBindingAsync, you can see it appears: 
![image](https://github.com/user-attachments/assets/70b614ba-6702-4489-8af4-7c966741dcac)

Reply:
It is shown when an error occurs during the TaskToPerform is executed. So if you throw an exception in the CheckForSharedBindingAsync, you can see it appears:

![image](https://github.com/user-attachments/assets/bb3f7cba-d61e-44bd-953f-f66087c6dc6c)